index.html

```diff
@@ -26,13 +26,13 @@
                 </li>
                 <li>
                     <a href="#" class="link">
-                        <img src="images/toolbar_icons/message.svg" alt="contact me">
+                        <img src="images/toolbar_icons/message.svg" alt="Contact author">
                     </a>
                 </li>
             </ul>
 
             <button type="button" aria-label="open close main menu" class="toolbar-icon">
-                <img src="images/toolbar_icons/hamburguer.svg" alt="">
+                <img src="images/toolbar_icons/hamburguer.svg" alt="Menu button">
```

Comment:
There is no need to include the words "button", "link", or "image of" in the alternative text. (Screen readers automatically provide that information 😎) Kindly make the necessary changes in order to meet the project requirements 🤓